tests/sentry/api/endpoints/notifications/test_notification_action_index.py

```diff
@@ -0,0 +1,228 @@
+from unittest.mock import patch
+
+from rest_framework import status
+
+from sentry.api.serializers.base import serialize
+from sentry.models.notificationaction import NotificationAction, NotificationActionProject
+from sentry.testutils import APITestCase
+from sentry.testutils.silo import region_silo_test
+
+NOTIFICATION_ACTION_FEATURE = ["organizations:notification-actions"]
+
+
+@region_silo_test(stable=True)
+class NotificationActionsIndexEndpointTest(APITestCase):
+    endpoint = "sentry-api-0-organization-notification-actions"
+
+    def setUp(self):
+        self.user = self.create_user("thepaleking@hk.com")
+        self.organization = self.create_organization(name="hallownest", owner=self.user)
+        self.other_organization = self.create_organization(name="pharloom", owner=self.user)
+        self.team = self.create_team(name="pale beings", organization=self.organization)
+        self.projects = [
+            self.create_project(name="greenpath", organization=self.organization),
+            self.create_project(name="dirtmouth", organization=self.organization),
+        ]
+        self.base_data = {
+            "serviceType": "slack",
+            "triggerType": "audit-log",
+            "targetType": "specific",
+            "targetDisplay": "@hollowknight",
+            "targetIdentifier": "THK",
+        }
+        self.login_as(user=self.user)
+
+    def test_requires_feature(self):
+        self.get_error_response(self.organization.slug, status_code=status.HTTP_404_NOT_FOUND)
+        self.get_error_response(
+            self.organization.slug,
+            status_code=status.HTTP_404_NOT_FOUND,
+            method="POST",
+        )
+
+    def test_get_simple(self):
+        notif_actions = [
+            self.create_notification_action(organization=self.organization),
+            self.create_notification_action(organization=self.organization),
+        ]
+        other_notif_action = self.create_notification_action(organization=self.other_organization)
+        with self.feature(NOTIFICATION_ACTION_FEATURE):
+            response = self.get_success_response(
+                self.organization.slug,
+                status_code=status.HTTP_200_OK,
+            )
+            assert len(response.data) == len(notif_actions)
+            assert serialize(other_notif_action) not in response.data
+            for action in notif_actions:
+                assert serialize(action) in response.data
+
+    @patch.object(
+        NotificationAction,
+        "get_trigger_types",
+        return_value=[(0, "teacher"), (1, "watcher"), (2, "beast")],
+    )
+    def test_get_with_queries(self, mock_trigger_types):
```

Comment:
Would be good to cover GET without a project filter honoring project permissions.